src/main/java/org/checkerframework/specimin/UnsolvedSymbolVisitor.java

```diff
@@ -71,6 +104,15 @@ public class UnsolvedSymbolVisitor extends ModifierVisitor<Void> {
    */
   private String chosenPackage = "";
 
+  /** This set has fully-qualified class names that come from jar files input */
+  private Set<@FullyQualifiedName String> classesFromJar = new HashSet<>();
+
+  /**
+   * This set has the fully-qualfied name of the synthetic return types created by this instance of
+   * UnsolvedSymbolVisitor
+   */
+  private Set<String> syntheticReturnTypes = new HashSet<>();
```

Comment:
I think this field (and the others in this class) can also be `final`. Adding `final` is the best practice in Java unless the field will be re-assigned, so we should get into the habit of doing so.